keps/sig-node/1370-cgroup-v2/kep.yaml

```diff
@@ -0,0 +1,21 @@
+title: cgroups v2
```

Comment:
It shouldn't be KEP number 1370.

You should open an issue in this repo, and the KEP number should be the number of that issue.

Reply:
thanks, opened a new issue and linked here